NEW-MEMBERS.md

```diff
@@ -0,0 +1,29 @@
+## New Members
+
+The purpose of this plan is to ensure that you become familiar with the team and
+know how you will contribute. The first step is to get yourself familiar with
+our mission at [Security TAG charter](governance/charter.md).
+
+New members are advised to:
+
+* Join the [CNCF Slack team](https://slack.cncf.io/), particularly
+  [#tag-security](https://cloud-native.slack.com/messages/CDJ7MLT8S) channel and
+  introduce yourself.
+* Initially go through the following documents in the repository:
+    * [README.md](https://github.com/cncf/tag-security/blob/main/README.md)
```

Comment:
Here and below: I'd suggest to use relative links rather than absolute. They are shorter and will work even after renaming a repo. For example:
```suggestion
    * [README.md](/blob/main/README.md)
```
But, please, double check that it works :)

Reply:
`/README.md` works -- agree that is a good idea!  I just renamed the file, but I'll go ahead and make that change while I've got it open in my editor :)

Reply:
fixed